api/api/util.py

```diff
@@ -23,13 +36,20 @@ def serialize(item):
         return item
 
 
-def _deserialize(data, klass):
-    """Deserializes dict, list, str into an object.
+def _deserialize(data: Union[dict, list, str], klass: type) -> object:
+    """Deserialize dict, list, str into an object.
 
-    :param data: dict, list or str.
-    :param klass: class literal, or string of class name.
+    Parameters
+    ----------
+    data : Union[dict, list, str]
```

Comment:
This syntax is not correct. In the docstrings, you use `or`.

Reply:
I have also reviewed all the parameters using the `Union` keyword.